package.json

```diff
@@ -76,6 +76,6 @@
     "folderify": "^0.6.0",
     "form-data": "^0.2.0",
     "har-validator-fsless": "^1.6.2",
-    "postman-collection": "1.2.8"
+    "postman-collection": "postmanlabs/postman-collection#develop"
```

Comment:
?

Reply:
we need a stable release of `postman-collection` to fix that thing.